views/map.ejs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Developers</title>
+</head>
+
+<body>
+	<h1>Name:</h1>
+	<form action="/map/get" method="POST">
+		<label for="name">First name:</label>
+		<input type="text" id="name" name="name" required>
+		<input type="submit" value="Submit">
+	</form><br><br>
+	<p>
+
+	</p>
+</body>
+
+</html>
```

Comment:
This is a cool way to test the route behavior without a REST client to test on, but since this app will be a faceless API we don't really need this.